presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java

```diff
@@ -209,6 +226,11 @@ long getRetryDelayInMs()
             return delayInMs + jitter;
         }
 
+        long getDuration()
```

Comment:
Where is this used? If needed, we should return the Duration, rather than returning a long with an unknown unit.